### ISSUE TITLE
Debian 9 INSTALL.md hints missing

### ISSUE THREAD

**Issue by aaronkaplan** [certtools/intelmq-manager#168]
Debian 8 differs in packages from Debian 9. Hence the instructions for Debian 9 break when trying to replicate the instructions for Debian 8.

```bash
# apt-get install git apache2 php5 libapache2-mod-php5
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Package php5 is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

Package libapache2-mod-php5 is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'php5' has no installation candidate
E: Package 'libapache2-mod-php5' has no installation candidate
```

**Comment by wagner-certat** [comment]
Do the instructions for Ubuntu 16.04 work?

**Comment by aaronkaplan** [comment]
> On 03 Sep 2018, at 15:40, Wagner <notifications@github.com> wrote:
> 
> Do the instructions for Ubuntu 16.04 work?

yes :)

**Comment by wagner-certat** [comment]
fixed by 1c8c081039e7e2b2cffc6e48a3bec05bf4e8eed4

**Comment by aaronkaplan** [comment]
ACK.
